migrate/scripts/object-id.js

```diff
@@ -0,0 +1,42 @@
+var mongodb = require("mongodb"),
```

Comment:
Should be `const`.  I don't think we use `mongodb` anywhere, could it be possible to do this instead?
```
const ObjectId = require('mongodb').ObjectId;
```